docs(faq): update content for SEO optimization MTA-1762 (#417)
scaleway/docs-content: faq/instances.mdx

---
meta:
  title: Instances

>   description: Discover Scaleway Instances.

Author: ```suggestion
  description: Discover Scaleway Instances and learn the difference between image and snapshot.
```

>   description: Discover Scaleway Instances and learn the difference between image and snapshot.

Reviewer: ```suggestion
  description: Discover Scaleway Instances and learn the difference between images and snapshots.
```

With the Standby option, you are still charged the same amount as if the instance is running.
</Message>

## Can I reach my Instance using a DNS hostname instead of its IP address?

> Yes, each instance has two DNS hostnames associated with it: 

Author: ```suggestion
Yes, each Instance has two DNS hostnames associated with it: 
```
